lib/di.js

```diff
@@ -57,7 +57,7 @@ export async function verifyPresentation({
     documentLoader: await createDocumentLoader({config}),
     suite: createSuites(),
     unsignedPresentation: !checks.includes('proof'),
-    checkStatus
+    checkStatus: _checkStatus
```

Comment:
Looks like this needs the same `checkStatus` wrapper as above in `verifyCredential`.  Probably need more presentation related tests.

Reply:
I could change it to that, but that would I think change the existing behavior. It's probably more consistent to do it that way and perhaps even a bug that it wasn't behaving that way before, so I'm just avoiding do so right now in this PR.